doc/source/whatsnew/v0.22.0.txt

```diff
@@ -326,7 +326,7 @@ Reshaping
 - Bug in :func:`DataFrame.stack` which fails trying to sort mixed type levels under Python 3 (:issue:`18310`)
 - Fixed construction of a :class:`Series` from a ``dict`` containing ``NaN`` as key (:issue:`18480`)
 - Bug in :func:`Series.rank` where ``Series`` containing ``NaT`` modifies the ``Series`` inplace (:issue:`18521`)
```

Comment:
rebase on master. can you move to 0.23 (docs were renamed), prob easiest to just check this file from master and past in new one

Reply:
Done. I also squashed some of my commits to clean up the commit history for you.